docs/pages/zencode-scenarios-qp.md

```diff
@@ -195,7 +195,89 @@ The result should look like:
 
 obviously this secret will be equal to Bob's secret.
 
+# Benchmark
 
+In the following we will compare the Quantum-Proof algorithms with the most common Elliptic-Curve algorithms (ECDSA, ECDH). All the results in the following are generated by the script [benchmark_sig.sh](https://github.com/dyne/Zenroom/blob/master/test/zencode_qp/benchmark_sig.sh) and [benchmark_enc.sh](https://github.com/dyne/Zenroom/blob/master/test/zencode_qp/benchmark_enc.sh).
+
+## Key length
+
+As first parameter we will look at the private and public keys length (measured in bytes).
+
+| Sizes      | private key | public key  |
+|------------|-------------|-------------|
+| ECDSA/ECDH | 32          | 65          |
+| Dilithium  | 2528        | 1312        |
+| Kyber      | 1632        | 800         |
+| NTRUP      | 1763        | 1158        |
+
+As we can see they Quantum-Proof keys are much longer. Will this affect the speed of the algorithms?
+
+## Signature
+
+We can divide the signature in four main parts: the generation of the *private key*, the generation of the *public key*, the *signature* and the *verification*. As can be seen from the tables below the Dilithium *key generation* is **not affected at all from the key length**, indeed both the key generation time (computed in microsecond) and memory (computed in Kibibyte) are not very different between Dilithium and ECDSA.
+
+| Algorithm | Key         | Time       | Memory |
```

Comment:
Missing unit for Time and Memory

Reply:
I tried it, but the μ symbol in the table header becomes an "M" using docsify and i have not found any solution to overcome this problem. For this reason I have wrote all the unit of measurement outside of the tables.

Reply:
I see, it gets transformed uppercase, you need some inline html, look at my commit

Reply:
Perfect, thank you 🙏